packages/@dcl/sdk-commands/src/commands/init/project.ts

```diff
@@ -0,0 +1,36 @@
+import path from 'path'
+
+import { SCENE_FILE } from '../../logic/scene-validations'
+import { Options } from '.'
+import { ScaffoldedProject } from './repos'
+import { Scene } from '@dcl/schemas'
+
+export function getMinimalSceneJson(): Partial<Scene> & { ecs7: boolean; runtimeVersion: string } {
+  return {
+    ecs7: true,
+    runtimeVersion: '7',
+    display: {
+      title: 'SDK7 Scene Template',
+      description: 'template scene with SDK7'
+    },
+    scene: {
+      parcels: ['0,0'],
+      base: '0,0'
+    },
+    main: 'bin/index.js'
+  }
+}
+
+export async function augmentProject(project: ScaffoldedProject, dest: string, options: Options) {
```

Comment:
I'll do it specifically to px. So i dont have to enter to that fn to see that it doesnt change anything if its a scene.
```suggestion
export async function createPxSceneJson(cwd: string, fs: Pick<CliComponents, 'fs'>) {
```